### ISSUE TITLE
testmybot is incompatible with recent versions of Botkit

### ISSUE THREAD

**Issue by Naktibalda** [codeforequity-at/testmybot#32]
I returned to my bot project after a long break and updated all dependencies.
Then I ran my tests and all `testmybot` tests failed.

I tried to identify the culprit and this is what I discovered:

Older versions of botkit fail with:
```
TestMyBot Test Suite for botkit-starter-web Ask FAQ question:
     AssertionError: object tested must be an array, a map, an object, a set, a string, or a weakset, but object given
      at compareObject (node_modules\@naktibalda\testmybot\lib\testbuilder.js:81:5)
```
https://github.com/codeforequity-at/testmybot/blob/master/testmybot/lib/testbuilder.js#L81
```js
assertCb(result, expected); 
```
I printed both variables with console.log() and this is what I got:
```
result 

{ message:
   { recipient: { id: 'me' },
     message: { text: 'expected message text' } } }

expected 

expected message text
```

The convo file in question is very simpy:
```
Test question

#me
question

#bot
expected message text
```

Newer versions fail with:
```
     Error: done() invoked with non-Error: nothing said in time
      at async.eachSeries (node_modules\testmybot\lib\testbuilder.js:51:17)
```

I manually found a commit which introduced this issue:
https://github.com/howdyai/botkit/commit/50fbf6ab6fcea06890c6559c797d04cca4bb0924

**Comment by Naktibalda** [comment]
Second incompatibility is caused by addition of
`facebook_botkit.middleware.spawn.use(function(worker, next) {` to
https://github.com/howdyai/botkit/blob/0b8e61d4c9b07a956c63bfb947dc82a52a8470d2/lib/Facebook.js#L369 which is overriding `controller.on('spawned', function(worker) {` in https://github.com/codeforequity-at/testmybot/blob/master/testmybot/helper/botkit.js#L20

**Comment by Naktibalda** [comment]
Second incompatibility was a bug of botkit.
@benbrown quickly [fixed](https://github.com/howdyai/botkit/commit/c88cdbd8f856c0732fbd97a3951d8a52319f134d) it and released 0.6.6.

**Comment by codeforequity-at** [comment]
Pull request merged to master!
